chainer/distribution.py

```diff
@@ -0,0 +1,420 @@
+import copy
+
+
+class Distribution(object):
+
+    """Interface of Distribution
+
+    `Distribution` is a bass class to treat probability distributions.
+
+    This class provides a mean to perform following operations.
+    1. Sampling random points.
+    2. Evaluating a functions about probability at given realization values.
+        (e.g., probability density function, probability mass function)
+    3. Obtaining properties of distributions.
+        (e.g., mean, variance)
+
+    Note that every method and property that computes them from
+    `chainer.Variable` can basically be differentiated.
+
+    In this class, sampled random points and realization values given in
+    function about probability is called "sample".  Sample consists of batches,
```

Comment:
Consider rewording "function about probability". How about "probability-related function"? or "a function related to probability"?